11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>This seems like the most important part in this tutorial.</p><p>Currently, it doesn't look having self-contained information so that people understand it enough by this.</p><p>How about adding more information like mathematical formulation? or the difference from the naive DQN?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>